lib/src/generator_utils.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:collection/collection.dart';
+import 'package:dartdoc/src/model/categorization.dart';
+import 'package:dartdoc/src/model/enclosed_element.dart';
+import 'package:dartdoc/src/model/indexable.dart';
+
+/// Convenience function to generate category JSON since different generators
+/// will likely want the same content for this.
+String generateCategoryJson(Iterable<Categorization> cateories, bool pretty) {
```

Comment:
Yes, this seems like a good idea.  However, categories is misspelled in parameter.

Reply:
Fixed